src/controllers/mainController.ts

```diff
@@ -307,6 +316,15 @@ export default class MainController implements vscode.Disposable {
 
                 Telemetry.sendTelemetryEvent('RunQuery');
 
+                if (execStatment) {
```

Comment:
@benrr101 I've refactored that code that to make it more clear what's going on.  When there is no selection range the editor selection property gives the cursor position, otherwise we use the start position in the selection as the cursor position.